src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php

```diff
@@ -202,6 +205,43 @@ private function addClassesSection(ArrayNodeDefinition $node)
                                             ->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('translation')
+                                        ->isRequired()
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->variableNode('option_value')->end()
+                                            ->arrayNode('classes')
+                                                ->isRequired()
+                                                ->addDefaultsIfNotSet()
+                                                ->children()
+                                                    ->scalarNode('model')->defaultValue(OptionValueTranslation::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('interface')->defaultValue(OptionValueTranslationInterface::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                                    ->scalarNode('repository')->cannotBeEmpty()->end()
+                                                    ->scalarNode('factory')->defaultValue(Factory::class)->end()
```

Comment:
@pjedrzejewski @lchrusciel  Why should  be `->cannotBeEmpty()`? In [Option Node Configuration](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php#L116) the factory hasn't  `->cannotBeEmpty()` and has `TranslatableFactory` and not normal factory because it is translatable resources. Even `optionValue` is translatable resources.

It's correct?